index.html

```diff
@@ -176,7 +176,31 @@ <h3 class="tool-title">Skills</h3>
                 </div>
             </div>
         </section>
-        
+
+        <!-- Contact form -->
+        <section class="website-form">
+            <p class="form-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            <form action="https://formspree.io/f/xqkonber" method="POST">
+                <label for="name"></label>
+                <input type="text" id="name" name="name" maxlength="30" class="imput-contact" placeholder="Full name" required>
+                <label for="mail"></label>
+                <input type="email" id="mail" name="email" class="imput-contact" placeholder="Email address" required>
+                <label for="msj"></label>
+                <textarea name="message" rows="5" maxlength="500" id="msj" placeholder="Hey I’d like to connect to..." required></textarea>
+                <button class="see-project" type="submit">Get in touch</button>
+            </form>
```

Comment:
- [ ] You have done so much great work on this project we are proud of you but there are ssomething you still need to address please kindly work more  on (Get in touch button) Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.Its very important to stick to figma design template to make your website look great please kindly fix this.